contents.json

```diff
@@ -16641,6 +16641,27 @@
       ],
       "date_added": "June 19 2018",
       "suggested_by": "@RMizin"
+    },
+    {
+      "title": "BookPlayer",
+      "category-ids": [
+        "media"
+      ],
+      "description": "Player for your DRM-free audiobooks",
+      "source": "https://github.com/TortugaPower/BookPlayer",
+      "itunes": "https://itunes.apple.com/us/app/bookplayer-audio-book-player/id1138219998",
```

Comment:
Could you remove 'us' from the itunes link?

Reply:
sure 👍